src/components/TxData.js

```diff
@@ -215,15 +215,20 @@ class TxData extends React.Component {
       });
     }
 
-    const renderDecodedScript = (decoded) => {
-      switch (decoded.type) {
+    const renderDecodedScript = (output) => {
+      switch (output.decoded.type) {
         case 'P2PKH':
         case 'MultiSig':
-          return renderP2PKHorMultiSig(decoded);
+          return renderP2PKHorMultiSig(output.decoded);
         case 'NanoContractMatchValues':
-          return renderNanoContractMatchValues(decoded);
+          return renderNanoContractMatchValues(output.decoded);
         default:
-          return 'Unable to decode';
+          let script = output.script;
+          try {
+            script = atob(output.script)
+          } catch {}
+
+          return `Unable to decode script: ${script.trim()}`;
```

Comment:
I feel we can't just do this because of injection attack vectors.

Reply:
@msbrogli It's already safe by design. If i want to put some HTML dynamically here, i have to do it explicitly.

Reply:
Ok. Great. Thanks!